loginsightwebhookdemo/__init__.py

```diff
@@ -100,7 +100,7 @@ def parseLI(payload, alert):
     })
     if ('Info' in payload and payload['Info'] is not None):
         alert.update({"info": payload['Info']})
-    elif ('messages' in payload and payload['messages'] is not None):
+    elif ('messages' in payload and len(payload['messages']) > 0):
```

Comment:
Earlier you suspected this might be `None`. Is that never true? Because you can't take the length of None.

```
>>> len(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: object of type 'NoneType' has no len()
```

If `payload['messages']` could be None, then you really need to check them all...

```
elif ('messages' in payload and payload['messages'] is not None and len(payload['messages']) > 0):
```

or consider that it's EAFP -- try to use it and handle the exception;

```
try:
   alert.update({"info": payload['messages'][0]['text']})
 except TypeError, IndexError, KeyError:
   alert.update({"info": ""})
```